### DISCUSSION TITLE
Where to run Stable Diffusion XL?

### DISCUSSION THREAD

**Discussion by Undinecat** [TheLastBen/fast-stable-diffusion#2583] [Q&A]
Hello, I'm new here. I used the Automatic1111 when Colab was free, I trained some models but I wanted to do something betterand then Colab stopped working for SD. I can't do this in Colab anymore, and I really want to work with my art in private (not using web servers who can steal my art style). I don't know if there's a free way to train a model, but I'm planing to change to Runpod or Paperspace, what would be better?, or buying a new computer is less expensive? Can you please give me some advice?.

**Comment by TheLastBen** [comment]
Runpod or Paperspace are pretty cheap you can explore them, very useful if you don't have a powerful computer
